### DISCUSSION TITLE
How to install podman v5.1.1 in old ubuntu such as 18.04 20.04

### DISCUSSION THREAD

**Discussion by lunaseazc** [containers/podman#22949] [Q&A] [answered]
Only source code install？ Or apt-get install ？

**Comment by afbjorklund** [comment, marked as answer]
You would have to install it from source code.

Could also run it in a VM, like on Mac or Win?

* https://github.com/containers/podman/discussions/22921

* https://github.com/containers/podman/discussions/17362
